components/quote-carousel/QuoteCarousel.style.js

```diff
@@ -71,7 +107,6 @@ const QuoteContainer = styled.div`
   left: 2em;
   bottom: 9em;
   margin-bottom: -12em;
-  overflow: hidden;
```

Comment:
question: just to double check we are "error" checking elsewhere that the quote is not too long?

Reply:
Yes will do a separate PR to make sure the quote carousel's error handling / spacing looks good on both about & story inquiry pages